actions/issue/file/action.yml

```diff
@@ -57,15 +56,19 @@ runs:
           exit 1
         fi
 
-        release_issue=$(gh issue list --repo "${{ inputs.repo }}" --state all --label "${{ inputs.label }}" --json number --jq .[0].number)
+        issue_data=$(gh issue list --repo paketo-buildpacks/dep-server --state all --label workflow-failure --json title --json number --jq .[0])
```

Comment:
```suggestion
        issue_data=$(gh issue list --repo "${{ inputs.repo }}" --state all --label "${{ inputs.label }}" --json title --json number --jq .[0])
```

Reply:
I think this jq search has a bit of a bug: Since we are only ever getting the 0th issue that has the specified label, we sometimes won't find the right issue that corresponds to the desired title. Imagine that `gh` query returns
```
[{ "number" : 2, 
  "title" : "Title B"
},
{ "number" : 2
  "title": "Title A"
}
]
```
and we're looking for the issue titled "Title A". We'll grab the 0th array element, but then fail the conditional on line 63 and skip ahead to filing a brand new issue.

Reply:
I think it'd be reasonable to filter the `gh` output based on title before grabbing the 0th element to achieve the desired outcome.